server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -40,6 +41,37 @@ public abstract class CICSOptionsCheckBaseUtility {
 
   private final List<SyntaxError> errors;
 
+  private static final Map<String, ErrorSeverity> SPECIAL_SEVERITIES =
+      new HashMap<String, ErrorSeverity>() {
+        {
+          put("ASIS", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("BUFFER", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("LEAVEKB", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("NOTRUNCATE", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("NOQUEUE", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("NOTRUNCATE", ErrorSeverity.WARNING);
+        }
+
+        {
+          put("TERMINAL", ErrorSeverity.WARNING);
+        }
+      };
```

Comment:
the following should work as well
```suggestion
      new HashMap<String, ErrorSeverity>() {
        {
          put("ASIS", ErrorSeverity.WARNING);
          put("BUFFER", ErrorSeverity.WARNING);
          put("LEAVEKB", ErrorSeverity.WARNING);
          put("NOTRUNCATE", ErrorSeverity.WARNING);
          put("NOQUEUE", ErrorSeverity.WARNING);
          put("NOTRUNCATE", ErrorSeverity.WARNING);
          put("TERMINAL", ErrorSeverity.WARNING);
        }
      };
```